### ISSUE TITLE
Voice Over issues in React Blob Explorer

### ISSUE THREAD

**Issue by JasonYeMSFT** [microsoft/AzureStorageExplorer#7172]
- [x] Voice Over doesn't announce the "(inferred)" part of the access tier in the data grid.
- [x] Voice Over announces "dimmed" for address bar and breadcrumbs.
